specs/Economics/README.md

```diff
@@ -69,15 +70,37 @@ Where `gasUsed[t] = sum([sum([gas(tx) for tx in chunk]) for chunk in block[t]])`
 Amount of `NEAR` on the account represents right for this account to take portion of the blockchain's overall global state. Transactions fail if account doesn't have enough balance to cover the storage required for given account.
 
 ```python
+def check_storage_cost(account):
+    accountLengthAmount = 0 if len(account_id) > 10 else ACCOUNT_LENGTH_BASELINE_COST / 3 ** (len(account_id) - 2)
```

Comment:
there seems to be a unit mismatch. `ACCOUNT_LENGTH_BASELINE_COST` is in yotcoNEAR while `3 ** (len(account_id) - 2)` computes the cost in NEAR

Reply:
Account length should not be part of storage cost, but instead part of the rent.

If we would switch to state staking, then account names have to be rethought as well. Larger stake is one option, but naming has to change.

Reply:
What is wrong with naming? `ACCOUNT_LENGTH_BASELINE_AMOUNT` ?

Reply:
@bowenwang1996  I don't understand your comment. `ACCOUNT_LENGTH_BASELINE_COST` in yoctoNEAR, the resulting `accountLengthAmount` is in yoctoNEAR, and it compares to account balance below in yoctoNEAR. `3 ** (len(account_id) - 2)` is just a rescaling.

Reply:
Nvm I got confused.